README/ReleaseNotes/v636/index.md

```diff
@@ -154,6 +154,7 @@ This release changes that behavior, meaning the `Scale(bool)` command argument i
     [RNTupleMergeOptions](https://root.cern/doc/v634/structROOT_1_1Experimental_1_1Internal_1_1RNTupleMergeOptions.html));
   * "rntuple.ErrBehavior=(Abort|Skip)": RNTuple-specific option that specifies the behavior of the RNTupleMerger on error (see link above);
   * "rntuple.ExtraVerbose": RNTuple-specific option that tells the RNTupleMerger to emit more information during the merge process.
+* An new, **experimental** entry iteration interface, the [RNTupleProcessor](https://root.cern/doc/v636/classROOT_1_1Experimental_1_1RNTupleProcessor.html), has been added. This interface provides the possibility to combine different RNTuples at runtime through *chains* and *joins*. As an effect, `RNTupleReader::OpenFriends()` has been removed.
```

Comment:
Wasn't the processor already there for 6.34?

Reply:
Yes, you're right 🙃. I'll change the note to better reflect the changes since then.